modules/factory.js

```diff
@@ -0,0 +1,51 @@
+var Factory = function(land)
+{
+	this.land = new Land();
+};
+
+Factory.prototype = {
+	createOrganism: function(land)
+	{
+		organismCounter = Util.getRandomInt(0,4);
```

Comment:
this should happen outside the factory.

ex.
You ask the factory to create shoes/t-shirt/pants.
You don't say "create me something" and factory creates w.e it wants.